src/liboslexec/llvm_instance.cpp

```diff
@@ -1620,8 +1620,7 @@ BackendLLVM::run()
     }
 
     if (use_optix()) {
-        std::string name = fmtformat("{}_{}", group().name(), group().id());
-        ll.ptx_compile_group(nullptr, name,
+        ll.ptx_compile_group(nullptr, group().name().c_str(),
```

Comment:
```suggestion
        ll.ptx_compile_group(nullptr, group().name().string(),
```

Extremely minor nit: I think this should be `group().name().string()` to return the string (which is part of the ustring internals, it doesn't make a new string). But if you say `.c_str()`, that'll give you the character pointer, which will then have to make another string to pass to ptx_compile_group().